example/rest-api/art-institute/art-institute.php

```diff
@@ -89,6 +89,51 @@ function register_aic_block(): void {
 		],
 	]);
 
+	$collection_query = HttpQuery::from_array([
+		'data_source' => $aic_data_source,
+		'endpoint' => function ( array $input_variables ) use ( $aic_data_source ): string {
+			$endpoint = $aic_data_source->get_endpoint();
+			return add_query_arg( [
+				'limit' => $input_variables['limit'],  
+				'fields' => 'id,title,image_id,artist_title',
+			], $endpoint );
+		},
+		'input_schema' => [
+			'limit' => [
+				'name' => 'Limit',
+				'type' => 'ui:input',
+				'default_value' => 10,
+			],
+		],
+		'output_schema' => [
+			'is_collection' => true,
+			'path' => '$.data[*]',
+			'type' => [
+				'id' => [
+					'name' => 'Art ID',
+					'type' => 'id',
+				],
+				'artist_title' => [
+					'name' => 'Artist Title',
+					'type' => 'string',
+					'path' => '$.artist_title',
+				],
+				'title' => [
+					'name' => 'Title',
+					'type' => 'string',
```

Comment:
Might be a good block to add some [`title`-type](https://github.com/Automattic/remote-data-blocks/pull/397) supports?

Reply:
Indeed 🥲 Nice catch.